boards/arm/v2m_musca_b1/v2m_musca_b1.yaml

```diff
@@ -0,0 +1,7 @@
+identifier: v2m_musca_b1
+name: ARM V2M MUSCA B1
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
+  - xtools
```

Comment:
We can add zephyr tools here due to the new version including v8m support

Reply:
Added - zephyr

Thanks